src/VirtoCommerce.Platform.Caching/GenericCrud/GenericCrudCachingRegion.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Primitives;
+using VirtoCommerce.Platform.Core.Caching;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.Platform.Caching.GenericCrud
+{
+    /// <summary>
+    /// Generic CRUD caching region implementation for use with generic crud service.
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public class GenericCrudCachingRegion<T> : CancellableCacheRegion<GenericCrudCachingRegion<T>> where T : Entity
```

Comment:
The current implementation is turned out  quite general, thereby I think we can rename `GenericCrudCachingRegion` -> `GenericCachingRegion` and move it to the https://github.com/VirtoCommerce/vc-platform/tree/master/src/VirtoCommerce.Platform.Caching

Reply:
OK